meeting_minutes/230119_SARIF_TC_72.md

```diff
@@ -0,0 +1,158 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company           | Role(s)                         |
+|:-----------|:----------|:------------------|:--------------------------------|
+| Adar       | Weidman   | JFrog             | Member.                         |
+| Aditya     | Sharad    | Microsoft         | Voting Member                   |
+| David      | Keaton    | Individual        | Chair                           |
+| David      | Malcolm   | Red Hat           | Guest                           |
+| Gerald     | Sullivan  | Micro Focus       | Voting Member                   |
+| Mary       | Martin    | Microsoft         | Voting Member                   |
+| Michael    | Fanning   | Microsoft         | Voting member                   |
+| Nathan     | Baird     | Microsoft         | Voting Member                   |
+| Paul       | Anderson  | Grammatech, Inc.  | Member                          |
+| Samuel     | Guyer     | Veracode          | Member                          |
+| Stefan     | Hagen     | Individual        | Secretary, taking notes         |
+| Thanassis  | Avgerinos | ForAllSecure Inc  | Voting Member                   |
+| Yekaterina | O'Neil    | Micro Focus       | Voting member                   |
+
+## 1.3 Procedures for this meeting (Co-Chair David)
+
+## 1.4 Approval of agenda (Co-Chair David)
+
+* https://www.oasis-open.org/committees/download.php/70642/agenda_20230119.html
+
+The agenda was approved.
+
+## 1.5 Approval of previous minutes (Co-Chair David)
+
+* [Minutes of 2022-12-08 Meeting #71](https://www.oasis-open.org/committees/document.php?document_id=70641&wg_abbrev=sarif)
+
+The minutes were approved.
+
+## 1.6 Review of action items and resolutions (Secretary Stefan)
+
+* ACTION on Michael to keep pushing to close on the wikipedia page
+  * ONGOING
+  * Action is up for adoption
+* ACTION on Jeff to provide an initial seed page at github to foster the future roadmap / extension discussion
+  * ONGOING
+  * Jeff send preliminary content to chair
+
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective members attending their first meeting
+
+* Adar Weidmann will achieve voting privs at the end of next meeting in case he joins
```

Comment:
As a new member, Adar achieves voting privileges on March 5, 60 days after he joined, because we had fewer than two meetings during that 60-day period.

This special rule is only for new members and people who have made a special request to use it.  (See section 1.4.4 of the following document.)
[https://www.oasis-open.org/policies-guidelines/tc-process-2017-05-26/](https://www.oasis-open.org/policies-guidelines/tc-process-2017-05-26/)